Test/WinDivert/WinDivertDeviceTest.cs

```diff
@@ -70,6 +70,15 @@ public void TestGetNextPacket()
             AssertTcp(capture);
         }
 
+        private void WebFetch()
+        {
+            var uriString = "http://1.1.1.1";
+
+            var webClient = new WebClient();
```

Comment:
don't forget to dispose with using

Reply:
yeah odd that the examples on the .net docs pages don't show usage of usage.